### DISCUSSION TITLE
NYT Opinion -  Is There a Crisis in Cosmology ?

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#139] [Cosmology Models]
[NYT opinion - Is there a crisis in cosmology?](https://www.nytimes.com/2023/09/02/opinion/cosmology-crisis-webb-telescope.html)

> *"Two serious issues with the standard model of cosmology would be concerning enough. But the model has already been patched up numerous times over the past half century to better conform with the best available data - alterations that may well be necessary and correct, but which, in light of the problems we are now confronting, could strike a skeptic as a bit too convenient."*

Too convenient indeed.

I received a .pdf copy the article from the ASSA.[^1]  I am sharing it here for those who do not have access to the New York Times.

[Opinion _ Is There a Crisis in Cosmology.pdf](https://github.com/a-cosmology-group/acg/files/12563337/Opinion._.Is.There.a.Crisis.in.Cosmology.pdf)

[^1]: Astronomical Society of Southern Africa

**Comment by ftessier** [comment]
This is a _fantastic_ essay! Thank you for sharing.

**Comment by Pierre314159** [comment]
Ethan is not going to be happy.

**Reply by RedshiftDrift** [reply]
You know him well!

Starts With A Bang — September 6, 2023
*The “crisis in cosmology” is pure exaggeration*
[https://bigthink.com/starts-with-a-bang/crisis-cosmology-exaggerated-lie/](https://bigthink.com/starts-with-a-bang/crisis-cosmology-exaggerated-lie/)

**Comment by sahil5d** [comment]
Have you reached out to the authors? They are quite aligned with ACG.

**Reply by Pierre314159** [reply]
Not IMO. With phrases like “ Edwin Hubble’s discovery in the 1920s that the universe was expanding “ and the possibility that “ the laws of physics can evolve and change over time “, they disqualify for ACG. Agreed, they made a giant step in the right direction, but the leap of faith needed is much greater.

It would be so simple to just consider the universe as not having had a beginning…

**Reply by RedshiftDrift** [reply]
Indeed @Pierre314159 is right.  I follow Adam Frank from mastodon, he would not want to join ACG.

This kind of article appears frequently, always giving hope to Big Bang deniers, but is forgotten the next day... [see discussion 140](https://github.com/orgs/a-cosmology-group/discussions/140#discussion-5610473)

**Comment by RedshiftDrift** [comment]
Eric Lerner strikes back
***Saying goodbye to the Big Bang***
*The prevailing story of our universe is starting to unravel *
[https://asiatimes.com/2023/09/saying-goodbye-to-the-big-bang/](https://asiatimes.com/2023/09/saying-goodbye-to-the-big-bang/)

**Reply by RedshiftDrift** [reply]
Eric Lerner's new video:
"Episode 8 Goodbye to the Big Bang"
[https://youtu.be/KIDMFgrM3us](https://youtu.be/KIDMFgrM3us)
with a promise to interview Dr. Rajendra Gupta of the University of Ottawa [about his article](https://academic.oup.com/mnras/article/524/3/3385/7221343); stay tuned.

**Reply by bgreenman** [reply]
Rajendra Gupta's paper might help mainstream scienc ease its way out of the Big Bang Concept, by minimizing controversy

**Comment by ExpEarth** [comment]
I think we can agree with the authors on the JWST findings, but is the Hubble Tension really a serious crisis for them?  For those of us who support a static model, how do we explain this divergence in values for the Hubble constant?

**Reply by bgreenman** [reply]
The NYTimes article on alternative cosmology theories gives important credability to the alternative cosmology movement.it makes it acceptable for certain elements of the establishment to support research, that before they would not support.

**Reply by RedshiftDrift** [reply]
@ExpEarth how do *you* explain this divergence in values for the Hubble constant?

**Reply by ExpEarth** [reply]
I can't account for it either.  But part of the reason is that I have never taken the CMB-based measurement of H too seriously, as it is based on false premises concerning the origin of the CMB.  I would be curious to hear your take on it, here or in other discussions about the Hubble Tension.

**Reply by RedshiftDrift** [reply]
My take on the CMB is the same as yours, the CMB is not the afterglow of the Big Bang.
My explanation was [discussed here](https://github.com/orgs/a-cosmology-group/discussions/58#discussioncomment-6310746) and earlier on the Gaggle.mail forum, and is based on the Stimulated-Transfer-Redshift (STR) model [described here](http://adsabs.harvard.edu/abs/2009ASPC..413..268M).  A new paper on STR is in the works (about 2/3 written); I'll post it here when it is published.

**Reply by ftessier** [reply]
I really enjoyed your 2009 asp paper, such clarity and conciseness! 👍🏻

**Reply by ExpEarth** [reply]
@RedshiftDrift But what about the Hubble Tension?  Do you suppose that it is a serious concern for the Big bang model?  Do you think a similar discrepancy could arise in your static model?  Can we assume you would favour the higher H value, as it is based on local observations?

My feeling is that the same tension could show up in a static model and so we should not rejoice at this tension in the BB model. 

Looking forward to seeing your published paper!

**Reply by HanDeBruijn** [reply]
>the CMB is not the afterglow of the Big Bang.

Agreed for sure. By the way, my take on the CMB is 3/3 written and has been published [here](https://github.com/orgs/a-cosmology-group/discussions/135#discussioncomment-6956930).
Summarizing. Solve the CMB temperature $\Theta$ from

$$
\frac{4\sigma}{c}\Theta^4 = \frac{(B.c.H)^2}{G} \quad \mbox{with} \quad B = \sqrt{\frac{3.\mbox{VMT}}{32\pi.W}} = 0.004170942703
$$

$c=$ lightspeed, $H=$ Hubble parameter, $G=$ gravitational constant, $4\sigma/c=$ radiation constant. Only four constants of nature (one with a tension) and a pure number $B$ with an order of magnitude of the [Fine-structure constant](https://en.wikipedia.org/wiki/Fine-structure_constant).

**Reply by mikehelland** [reply]
Hubble's constant isn't actually what's being measured. The time dilation of supernovae and their distance is measured. When one tries to fit LCDM to those measurements, the model's best fist is H0 = 73.0 km/s/Mpc. When aspects of the CMB are measured, fitting LCDM to that data says H0 = 67.4 km/s/Mpc.

In a static universe, the luminosity distance (dL) is a different function than in LCDM. I have dL = c/H0 (1 + z) log(1 + z), which gives a best fit to the supernovae data at H0 = 70.5 km/s/Mpc.

The question then is, in this static universe, what is the CMB?

**Reply by RedshiftDrift** [reply]
@mikehelland 
> The question then is, in this static universe, what is the CMB?

The CMB is from [photons reaching an equilibrium between redshift and the Sunyaev-Zel'dovich effect](https://github.com/orgs/a-cosmology-group/discussions/58#discussioncomment-6310746) As mentioned above this was discussed in more details last year when ACG used a private discussion forum, so I am not going to repeat the argument here.

> In a static universe, the luminosity distance (dL) is a different function than in LCDM

Interestingly the two functions are very similar.  Plot (in Hubble distance units)

$$ D_L^{static} = (1+z) ln(1+z) $$

and

$$ D_L^{LCDM} = (1+z) \int_0^z \frac{{\rm d}z'}{\sqrt{\Omega_m(1+z')^3 + \Omega_\Lambda}} $$

with $\Omega_m = 0.286$ and $\Omega_\Lambda = 0.714$, up to redshift $z \simeq 6$, and the curves overlap quite well.

**Reply by mikehelland** [reply]
That is the luminosity distance - redshift relationship in my pet hypothesis. 

Is that the same one you use? 

A couple other questions, that I hope are no inappropriate?

I'm not familiar with this group. Is there a "consensus" model promoted here? Or more of a non-expanding free for all?

Are you adding TEX to the post or attaching images?

**Reply by RedshiftDrift** [reply]
@mikehelland 
> I'm not familiar with this group.

Please familiarize yourself with the group by exploring [the website (click here)](https://cosmology.info/), read the last page of one [Newsletter](https://cosmology.info/newsletters/), read the [Cosmology Statement](https://cosmology.info/org/open-letter-on-cosmology.html) and the [discussions Welcome notice](https://github.com/orgs/a-cosmology-group/discussions/15#discussion-5258529).  Have a look at the [discussions](https://github.com/orgs/a-cosmology-group/discussions) and select "Filter:All" to see all of them.

> Are you adding TEX to the post or attaching images?

Yes, there is an example of how to add tex to a post in the [Welcome notice](https://github.com/orgs/a-cosmology-group/discussions/15#discussion-5258529) and a link to the Markdown Syntax page that explains how to attach images.

> Is there a "consensus" model promoted here? Or more of a non-expanding free for all?

**No** *consensus model* is promoted here.  Any model can be discussed as long as you abide by the [Code of Conduct](https://github.com/a-cosmology-group/acg/blob/main/CODE_OF_CONDUCT.md) and the [Guidelines](https://github.com/a-cosmology-group/acg/blob/main/CONTRIBUTING.md)

Most discussions are about the problems of the Big Bang model, but there are as many solutions as there are participants 😄 some of them are [listed here](https://cosmology.info/models/) but not limited to this list!

**Reply by RedshiftDrift** [reply]
> That is the luminosity distance - redshift relationship in my pet hypothesis.
> Is that the same one you use?

Yes, that's the luminosity-distance I use for supernovae. I use $D_{L \ ga} = \sqrt{1+z} \ ln(1+z)$ for galaxies and $D_A = ln(1+z)$.

I'll read about [your algorithm](https://github.com/orgs/a-cosmology-group/discussions/155#discussion-5678351) and comment on it... a bit later.

**Reply by HanDeBruijn** [reply]
@mikehelland .
>some of them are listed here but not limited to this list!

One of the participants not in _this list_ is a strong proponent of [**Halton Arp's Ideas**](https://github.com/orgs/a-cosmology-group/discussions/141#discussioncomment-6966770).

**Reply by HanDeBruijn** [reply]
@mikehelland .
>The CMB is from [photons reaching an equilibrium between redshift and the Sunyaev-Zel'dovich effect](https://github.com/orgs/a-cosmology-group/discussions/58#discussioncomment-6310746)

Not that I know of. The CMB is explained by Halton Arp's [Variable Mass Theory](https://www.youtube.com/watch?v=T7jVviVBzKs) together with straightforward conversion of the gravitational self energy of the observable universe into heat radiation: [**Let there be CMB**](http://www.alternatievewiskunde.nl/cosmology/CMBR.htm).

**Comment by bgreenman** [comment]
I think its times to test Alternative explanations of CMB's origin, but the
main obstacle is finding funding

**Reply by HanDeBruijn** [reply]
>the main obstacle is finding funding

Funding? I would say, act like Albert Einstein and become an [assistant at the local patent office](https://en.wikipedia.org/wiki/Albert_Einstein#1902%E2%80%931909:_Assistant_at_the_Swiss_Patent_Office).
It's easy. You can do the work you _must_ do with one finger in your nose, say within half a day.
And then you can use the rest of the day and the other half of your brain for doing the real thing.
No funding needed at all. After one [Annus mirabilis](https://en.wikipedia.org/wiki/Annus_mirabilis_papers) at the office you're there.

**Reply by bgreenman** [reply]
One possibility could be the quantum world, i read about particles coming in and out of our visible universe, it could be a way that matter is recycled and can have several layers-dimensions of its own. a deeper investigation might find out many surprises

**Reply by HanDeBruijn** [reply]
Sure. Why not make up more mysteries than we already have.

**Reply by bgreenman** [reply]
I think some our current mysteries can be solved without making more up, only a few new ones would be found since 21st century instermentscould not go that deep
